### PR TITLE
[test] Remove unused code

### PR DESCRIPTION
**Pull request by maflcko**

### CODE REVIEW

File: src/test/util_tests.cpp [hunk 1]
```diff
@@ -244,7 +244,6 @@ BOOST_AUTO_TEST_CASE(util_IsHex)
 BOOST_AUTO_TEST_CASE(util_seed_insecure_rand)
 {
     int i;
```
Comment (paveljanik) [inline src/test/util_tests.cpp]: This `i` can be moved down to `for` cycle too.

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/8578/commits/fafb4ae498db607a9f125792ac6fc4fe9fe4c053

**Comment by btcdrak** [comment]
ACK fafb4ae

**Comment by jonasschnelli** [comment]
utACK https://github.com/bitcoin/bitcoin/pull/8578/commits/fa1cf9e7b83c7d1ac6130589c24c055406ba407a

**Pull request merged**